### PR TITLE
Fixed so that all PHP scripts are executed as www-data instead of root

### PR DESCRIPTION
**Pull request by vidarl**
In order to make this a bit more secure, this PR changes so that:
 - PHP scripts are now executed as ez user

It also ensures that the .php files in the application itself are owned by ez, while www-data user has write access to folders on a need-to basis

### CODE REVIEW

**Comment by vidarl** [comment]
@andrerom : ready for review

**Comment by vidarl** [comment]
@andrerom : Updated PR, now created user ez

File: docker-compose_ezpinstall.yml [hunk 1]
```diff
@@ -2,16 +2,18 @@ ezpublishvol:
   image: debian:jessie
   volumes:
    - ./volumes/ezpublish:/var/www
-  # Remove .keep file so composer is able to install in the directory afterwards
-  command: rm /var/www/.keep
+  # Remove .keep file so composer is able to install in the directory afterwards. UID=10000 and GUID=10000 is ez user in ezphp image
+  command: bash -c "rm /var/www/.keep; chown 10000:10000 /var/www"
```
Comment (andrerom) [inline docker-compose_ezpinstall.yml]: hmm, this is complicating this a bit.

**Comment by andrerom** [comment]
+1

**Pull request merged**